lxd/storage_lvm_utils.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/lxc/lxd/lxd/rsync"
 	"github.com/lxc/lxd/lxd/state"
 	driver "github.com/lxc/lxd/lxd/storage"
+	"github.com/lxc/lxd/lxd/storage/drivers"
```

Comment:
@monstermunchin please can you import this using the alias `storageDrivers` so it is consistent with the other uses of this package. Ideally the `storage` package itself would also be imported as `storagePools` but as this is the legacy code base I've left the old `driver.x` references alone. :)